devcluster/devcluster.py

```diff
@@ -10,12 +10,28 @@
 _save_cursor = None
 
 
+def tput(args):
+    p = subprocess.Popen(["tput", *args], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    out, err = p.communicate()
+
+    if p.returncode != 0:
+        err_str = err.decode('utf-8').strip()
+        err_msg = str(f"'{' '.join(['tput', *args])}' "
+                      f"exited non-zero({p.returncode}):\n"
+                      f"stdout: '{out.decode('utf-8').strip()}'\nstderr: '{err_str}'\n")
+        if 'unknown terminal' in err_str:
+            err_msg += str("If using a conda environment you may need to copy "
+                           "your '/usr/share/terminfo/*/<term_name>' into "
+                           "'/path/to/conda/envs/<ENV>/share/terminfo/*/<term_name>")
```

Comment:
Can we suggest that `conda` users `export TERMINFO=/usr/share/terminfo` instead?  That works for urxvt on my machine.